File: poker-client/src/app/api/models/draw-game-draw-model.ts
/* tslint:disable */
import {CardModel} from './card-model';

/**
 * Model representing data in the draw game.
 */
export interface DrawGameDrawModel {
  card?: CardModel;

  /**
   * Flag that is true if this draw won the hand.
   */
  winner?: boolean;
}

File: poker-server/src/main/java/com/poker/poker/models/game/DrawGameDrawModel.java
package com.poker.poker.models.game;

import io.swagger.v3.oas.annotations.media.Schema;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Schema(description = "Model representing data in the draw game.")
public class DrawGameDrawModel {

  @Schema(description = "Card that was drawn.", implementation = CardModel.class)
  private CardModel card;

  @Schema(description = "Flag that is true if this draw won the hand.", example = "true")
  private boolean winner;

  @Schema(description = "Flag that is true if this draw is next.", example = "true")
  private boolean acting;
}
